modules/vba_documents/app/serializers/vba_documents/upload_serializer.rb

```diff
@@ -17,6 +17,12 @@ def scrub_unnecessary_keys(pdf_hash)
         pdf_hash.delete(DOC_TYPE_KEY.to_s)
         pdf_hash.delete(SOURCE_KEY.to_s)
         pdf_hash.delete('submitted_line_of_business')
+
+        if pdf_hash['content'].present?
+          pdf_hash['content'].delete('sha256_checksum')
+          pdf_hash['content']['attachments']&.each { |attach_hash| attach_hash.delete('sha256_checksum') }
```

Comment:
Why are we deleting these keys?

Reply:
@cilestin The thought behind deleting them was so that we don't return them in the `show` response, since they're meant for our internal use and probably don't provide any use to the consumer (who submit a multipart payload rather than individual documents). We already return an MD5 checksum for the full submission payload in our response to the `PUT` endpoint, which the consumers can use to verify their submission receipt.